### ISSUE TITLE
microRNA prediction Tool

### ISSUE THREAD

**Issue by SebastianHoernstein** [bgruening/galaxytools#173]
Would be happy if that tool could be integrated :-)

http://omictools.com/targetfinder-s7192.html

**Comment by yhoogstrate** [comment]
This looks like an interesting piece of software, indeed. I was browsing the code and when I read the COPYRIGHT file, it says:

```
Copyrite
========

 This software, including but not limited to the code and use, is under the standard Oregon State University Copyright.
 Please do NOT distribute this code or information.

 "Copyright 2010 Oregon State University"


 NOTE:
 Non-commercial use is allowed under the above copyright. This distribution is not for commercial applications.
 Please contact targetfinder@cgrb.oregonstate.edu for commercial purposes and copyright information.
```

I am not sure if such license allows (re-)distribution via a toolshed because of the scentence ``` Please do NOT distribute this code or information.```.

**Comment by Stortebecker** [comment]
I agree that miRNA binding prediction would be a nice functionality. There are tons of free (or, I think so) software that could be integrated. Maybe to start with: 

http://www.targetscan.org/
http://www.microrna.org/

**Comment by bgruening** [comment]
@al2na is this something your lab would be interested to work on? PicTar etc ..?

**Comment by al2na** [comment]
we have targetScan and pictar in dorina, users can access the targets for multiple species. microRNA.org is quite old but we can also include them to Dorina and serve it via there

**Comment by bgruening** [comment]
@al2na yeah this is great. But do you think we should offer both tools as standalone predictions tools as well?

@SebastianHoernstein @Stortebecker Would dorina be enough for your use cases? Dorina is already integrated in FR-Galaxy.

**Comment by SebastianHoernstein** [comment]
hi,

I have no experience with dorina. I have used TargetFinder
directly on the commandline and also integrated predictions
of psRNAtarget (web application). Both are among several
tested (Srivastava et al., 2014 BMC Genomics 348, also my
experience when tested against other tools) the best
performing tools in every category. Thus, I can only
recommend TargetFinder. I´m not sure if this was helpfull
to you. From my part I´m currently finished with miRNA
target predictions.
@björn: I forgott my Galaxy password... could you help me?

thanks and good luck!

Sebastian

**Comment by al2na** [comment]
I will put it on the to do list, i looked at targetFinder doesn't seem to be complicated to integrate.

**Comment by bgruening** [comment]
@al2na awesome thanks!
@SebastianHoernstein @erxleben is taking care of your problem :)

**Comment by bgruening** [comment]
@SebastianHoernstein we have now [miranda](http://cbio.mskcc.org/microrna_data/manual.html) on our server. Maybe this is interesting for you.

@al2na any update on the targetFinder front?

**Comment by nsoranzo** [comment]
@bgruening Are you using https://toolshed.g2.bx.psu.edu/view/tgac/miranda/ ?

Also, I'm working on a TargetScan wrapper for Galaxy, if you are interested.

**Comment by bgruening** [comment]
Yes, exactly this tools :)
TargetScan and PicTar would be nice indeed!

**Comment by yhoogstrate** [comment]
this got just added to the main tool shed: https://toolshed.g2.bx.psu.edu/view/earlhaminst/miranda

**Comment by bgruening** [comment]
thanks @yhoogstrate !
